config/checkstyle_checks.xml

```diff
@@ -417,6 +417,13 @@
       <property name="query" value="//ANNOTATION[./IDENT[@text='Issue']]"/>
       <message key="matchxpath.match" value="Avoid using @Issue annotation."/>
     </module>
+    <!-- until https://github.com/checkstyle/checkstyle/issues/5234 -->
+    <module name="MatchXpath">
+      <property name="id" value="MatchXPathBranchContains"/>
+      <property name="query" value="//DOT//IDENT[@text='branchContains']"/>
```

Comment:
Just curious why `DOT` and not `METHOD_CALL` or something similar?

Reply:
Good point. Fixed.